app.js

```diff
@@ -50,26 +50,40 @@ class UI {
   static addBookToList(book) {
     const list = document.querySelector('#book-list');
 
-    const lItem = document.createElement('li');
+    const row = document.createElement('tr');
 
-    lItem.innerHTML = `
-      <p>${book.title}</p>
-      <p>${book.author}</p>
-      <p>${book.id}</p>
-      <p><a href="#">Remove</a></p>
+    row.innerHTML = `
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td>${book.id}</td>
+      <td><a href="#" class="btn btn-outline-primary btn-sm">Remove</a></td>
```

Comment:
- [ ] For each row that is used to display the books you can remove the id since it's not part of the given design. Also, when you look carefully, the title and the author are joined with a `by` prefix. Please fix this.
![image](https://user-images.githubusercontent.com/68167810/124962229-0d837f00-e027-11eb-99df-f50b7796cb19.png)
- [ ] The remove button should also be placed towards the end of the row. There is currently some space towards the right.